### PR TITLE
Consolidate WebSDK into SDK

### PR DESCRIPTION
**Pull request by mmitche**
Same PR as before, but with history preserved instead.

### CODE REVIEW

**Comment by mmitche** [comment]
@dsplaisted @vijayrkn Alright, looking solid over here.

**Comment by wli3** [comment]
Could you add [codeowner](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) to Vijay's team for websdk files?

**Comment by mmitche** [comment]
> Could you add [codeowner](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) to Vijay's team for websdk files?

@vijayrkn @wli3 Do you know what team that would be? There's nothign obvious in the current websdk repo.

**Comment by vijayrkn** [comment]
We can just start with me & I can get devs added as needed.

**Comment by mmitche** [comment]
@wli3 Done.

**Comment by vijayrkn** [comment]
@mmitche - The changes looks great. Do you happen to have link to the previous run so that we can verify if the tests are getting run as part of this.

**Review by vijayrkn — APPROVED** [review]

**Comment by vijayrkn** [comment]
> @mmitche - The changes looks great. Do you happen to have link to the previous run so that we can verify if the tests are getting run as part of this.

Verified & looks good.

**Comment by mmitche** [comment]
https://dev.azure.com/dnceng/public/_build/results?buildId=673531&view=results

Should have not run End2End tests like the previous state of websdk was

File: src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>net472;$(SdkTargetFramework)</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">net5.0</TargetFrameworks>
+    <AssemblyName>Microsoft.NET.Sdk.Publish.Tasks.Tests</AssemblyName>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <EndToEndTestsDisabled>true</EndToEndTestsDisabled>
```
Comment (vijayrkn) [inline src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj:7]: This is what is causing the E2E tests to be disabled.

Comment (vijayrkn) [reply src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj:7]: Let me try running it locally to see if there are any issues, else we can enable it. I had disabled it for the net5.0 migration.

Comment (mmitche) [reply src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj:7]: Yep, just matching your change. May want to get this in first, then reenable.

Comment (vijayrkn) [reply src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj:7]: yup. Agreed. hence the PR approval.

**Pull request merged**